src/app/util/dataFormatter.service.js

```diff
@@ -30,16 +31,38 @@ angular.module('util').factory('dataFormatter', ['util', 'i18n', function(util,
 	 * been fetched.
 	 */
 	var getValueHtml = function(datavalue, numPropId, properties, missingTermsListener, inline, short) {
+		function sanitize(str, forURI) {
+			if (angular.isUndefined(str)) {
+				return;
+			}
+
+			if (angular.isUndefined(forURI)) {
+				forURI = false;
+			}
+
+			var decoded = he.decode(str, { isAttributeValue: forURI });
```

Comment:
I don't quite understand why the decode is needed here and how this can be faithful wrt. the original string as provided by Wikidata